sdk/src/Core/Amazon.Runtime/CredentialManagement/SharedCredentialsFile.cs

```diff
@@ -463,6 +491,17 @@ public void UnregisterProfile(string profileName)
             _credentialsFile.Persist();
         }
 
+        /// <summary>
+        /// Deletes the sso-session section with the given name from the SharedCredentialsFile, if one exists.
+        /// </summary>
+        /// <param name="ssoSessionName">The name of the sso-session section</param>
+        public void UnregisterSsoSession(string ssoSessionName)
+        {
+            Refresh();
+            _credentialsFile.DeleteSection("sso-session" + " " + ssoSessionName);
```

Comment:
Is the SEP for config strict in that there can be only one space or can you have any whitespace between the prefix and name?